ambari-server/pom.xml

```diff
@@ -1575,7 +1575,7 @@
     <dependency>
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
-      <version>9.3-1101-jdbc4</version>
+      <version>${postgres.version}</version>
```

Comment:
`<version>` can be omitted, it's inherited from the parent pom.

Reply:
Oh, nice! I'll make the change.